taccsite_cms/static/site_cms/css/src/_imports/trumps/s-breadcrumbs.css

```diff
@@ -0,0 +1,62 @@
+/*
+Breadcrumbs
+
+A list of links and to ancestor navigation (pages or sections that are a parent, grandparent, et cetera of the current page or section).
+
+Notice: A scope class is used to enforce appropriate semantic HTML.
+
+Markup:
+<!-- major navigation -->
+<nav class="s-breadcrumbs">
+  <ol>
+      <li><a href="/"><span>Home</span></a></li>
+      <li><a href="/"><span>Science</span></a></li>
+      <li><span>Publications</span></li>
+  </ol>
+</nav>
```

Comment:
__Notes__
- By setting the class on the `<nav>`, anything else in the navigation will get [global breadcrumb styles](https://github.com/TACC/Core-CMS/pull/311/files#diff-4ddea17c5e5fa8849a9179e180edc0c9f790940473c77f6f44a2d47b1670146fR34-R40).
- [Why `<nav>`?](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#usage_notes)

Reply:
```suggestion
<!-- major navigation -->
<!-- SEE: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#usage_notes -->
<nav class="s-breadcrumbs">
  <ol>
      <li><a href="/"><span>Home</span></a></li>
      <li><a href="/"><span>Science</span></a></li>
      <li><span>Publications</span></li>
  </ol>
</nav>
```

Reply:
I have added this information into the code as a comment (adef3a6).